packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -415,3 +415,20 @@ class RenderingFlutterBinding extends BindingBase with GestureBinding, ServicesB
     renderView.child = root;
   }
 }
+
+/// Client for listening to system fonts change.
```

Comment:
"Mixin for [RenderObject] that will call [markNeedsLayout] whenever the system fonts change".

Can you also add an explanation why/when a developer should mix this in?